keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: New CPUManager Static Policy which spread hyperthreads across physical CPUs to better utilize CPU Cache
+kep-number: 4176
+authors:
+  - "@Jeffwan"
+  - "@LingyanYin"
+  - "@horacexd"
+  - "@LastNight1997"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: "2023-09-04"
+reviewers:
+  - "@ffromani"
+approvers:
+  - "@sig-node-tech-leads"
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
+  beta: "v1.31"
+  stable: "v1.33"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: "CPUManagerPolicyAlphaOptions"
+    components:
+      - kubelet
```

Comment:
Does the kube-apiserver validate the names of CPU manager policy options? If so, this gate is relevant there too.

Reply:
kubelet does this validation atm, I don't think this will change

Reply:
It does not as far as I know.

Here are three KEPs similar to this one that introduce new policy options in the same way: 
* https://github.com/kubernetes/enhancements/pull/3334
* https://github.com/kubernetes/enhancements/pull/2904
* https://github.com/kubernetes/enhancements/pull/2626